.pre-commit-config.yaml

```diff
@@ -42,7 +42,7 @@ repos:
     args: [--max-line-length=88, --select=C,E,F,W,B,B950, --extend-ignore = E203,E501]
     types_or: [python, cython]
 - repo: https://github.com/pre-commit/mirrors-clang-format
-  rev: v16.0.5
+  rev: v18.1.3
```

Comment:
This is the version of clang-format that gets installed via `apt install clang-format` on ubuntu 24.04.